packages/core/admin/admin/src/translations/en.json

```diff
@@ -91,6 +91,13 @@
   "Settings.apiTokens.title": "API Tokens",
   "Settings.apiTokens.types.full-access": "Full access",
   "Settings.apiTokens.types.read-only": "Read-only",
+  "Settings.apiTokens.duration.7-days": "7 days",
+  "Settings.apiTokens.duration.30-days": "30 days",
+  "Settings.apiTokens.duration.90-days": "90 days",
```

Comment:
Do we already use any modules for converting a length of time to localized words? If this could be built to convert any "lifespan in ms" to a human readable length of time, that would give us a lot more flexibility in extending this.